### PR TITLE
Fix header massiveness

### PR DESCRIPTION
**Pull request by rgarner**
[BUG: have introduced header issues with rebase of PR #90](https://trello.com/c/OLUw0toi/161-bug-have-introduced-header-issues-with-rebase-of-pr-90)

Fix header nonsense introduced in #90:

screen shot 2017-09-14 at 10 33 45

### CODE REVIEW

File: app/views/research_sessions/data.html.erb [hunk 1]
```diff
@@ -4,7 +4,11 @@
   <h1 class="subtitle-large">Data</h1>
   <%= barnardos_form_with model: @research_session, url: wizard_path do |form| %>
 
-    <%= form.radio_group_vertical :shared_with, SharedWith::NAME_VALUES %>
+    <%= form.radio_group_vertical \
+        :shared_with,
+        SharedWith::NAME_VALUES,
+        legend: 'Who will the data be shared with?'
```
Comment (ztolley) [inline app/views/research_sessions/data.html.erb]: This can't come from labels?

File: app/views/research_sessions/expenses.html.erb [hunk 2]
```diff
@@ -5,30 +5,15 @@
   <%= barnardos_form_with model: @research_session, url: wizard_path do |form| %>
 
       <%= form.labelled_text_field :travel_expenses_limit,
-            label_options: {
-              hint: 'If you allow travel expenses, what is the maximum allowed?'
-            },
-            text_options: {
-              placeholder: '0.00'
-            }
+            text_options: { placeholder: '0.00' }
```
Comment (ztolley) [inline app/views/research_sessions/expenses.html.erb]: Where did the hint go?

File: config/locales/en.yml [hunk 3]
```diff
@@ -46,9 +46,9 @@ en:
     other_recording_method: "What is the other recording method?"
     duration: "How long will the session be? (optional)"
     participant_equipment: "What do participants need to bring? (optional)"
-    travel_expenses_limit: "Travel expense limit"
-    food_expenses_limit: "Food expense limit"
-    other_expenses_limit: "Other expense limit"
+    travel_expenses_limit: "If you allow travel expenses, what is the maximum allowed?"
```
Comment (ztolley) [inline config/locales/en.yml]: oh here it went.. hmm

File: app/views/research_sessions/expenses.html.erb (hunk 2 above)
Comment (rgarner) [reply app/views/research_sessions/expenses.html.erb]: It's gone after talking to @sl33p

Comment (rgarner) [reply app/views/research_sessions/expenses.html.erb]: Well, it became the label.

File: app/views/research_sessions/data.html.erb (hunk 1 above)
Comment (rgarner) [reply app/views/research_sessions/data.html.erb]: No, because it isn't one :)

But yeah, this needs some thought.

Comment (ztolley) [reply app/views/research_sessions/data.html.erb]: Even though it isn't a label tag it's still a label or is it strict and only get the label for the option. How did it get it before the change ?

Comment (rgarner) [reply app/views/research_sessions/data.html.erb]: It's always used the `legend:` keyword. It needs some thought. Like I say, this PR is just about fixing the headers I broke, though.

**Pull request merged**